core/model/src/payment.rs

```diff
@@ -200,11 +200,37 @@ pub mod local {
         pub outgoing: StatusNotes,
         pub incoming: StatusNotes,
     }
+
+    #[derive(Clone, Debug, Serialize, Deserialize, Default)]
+    pub struct StatValue {
+        pub value: BigDecimal,
+        pub count: u64,
```

Comment:
```suggestion
        pub total_amount: BigDecimal,
        pub agreements_count: u64,
```
I could suggest a small renaming of these fields to make explicit what they track. `value` and `count` are pretty generic names.

Reply:
now:
```$ yagna payment status
---
amount: "1001.67860972032465585"
incoming:
  accepted:
    count: 89
    value: "2.81057283828077799"
  confirmed:
    count: 82
    value: "2.58730856099244463"
  requested:
    count: 131
    value: "3.13321155406238910144"
outgoing:
  accepted:
    count: 14
    value: "0.894242557244733360"
  confirmed:
    count: 14
    value: "0.894242557244733360"
  requested:
    count: 73
    value: "2.513629358149274436700"
reserved: "129.72759754410611109"
```

after change:

```
amount: "1001.67860972032465585"
incoming:
  accepted:
    agreements_count: 89
    total_amount: "2.81057283828077799"
  confirmed:
    agreements_count: 82
    total_amount: "2.58730856099244463"
  requested:
    agreements_count: 131
    total_amount: "3.13321155406238910144"
outgoing:
  accepted:
    agreements_count: 14
    total_amount: "0.894242557244733360"
  confirmed:
    agreements_count: 14
    total_amount: "0.894242557244733360"
  requested:
    agreements_count: 73
    total_amount: "2.513629358149274436700"
reserved: "129.72759754410611109"
```

camel case ?

Reply:
@prekucki 'after change' looks fine for me. It's clearer what the values mean. We can ask @83tb or someone from SDK team what do they think.

Reply:
@Wiezzel I can agree with your observation 👍

Reply:
I would even change `amount` to `amount_of_funds_held`, `incoming` to `incoming_payments`, `reserved` to `reserved_funds`. But that's me, I prefer names that are self-explanatory.

Reply:
@filipgolem `amount_of_funds_held` is bit too baroque. How about `account_balance`?

Reply:
Sounds good to me. I'm only not 100% sure if "balance" (as used in banking) includes funds that are "reserved".